### ISSUE TITLE
Option to configure default colors

### ISSUE THREAD

**Issue by nyonson** [jtheoof/swappy#131]
Pretty specific request here, but would it be possible to expose configuration options for the default red/blue/green colors? I would like to match them to color palettes I work with often.

**Comment by voronind-com** [comment]
Yes! This is ancient, but I hope it'll get implemented.

**Comment by physicophilic** [comment]
Yes, looking for this exactly. There is an old related PR too @jtheoof

**Comment by voronind-com** [comment]
Sadly even tho the PR got merged we still need to hit C key to use it. Was kinda hoping to have a custom color by default. Also OP asked for changing r/g/b values, not custom color.

**Comment by monday-j** [comment]
Hey, I wanted to add to this as well, the merge in #135 appears to allow us to set the _custom_ colour but this issue in particular is to request changing the default RGB values or perhaps being able to define a custom palette.

From a configuration perspective, I'd imagine we could set the following values:

```
red_color=rgba(238, 22, 31, 1)
blue_color=rgba(0, 66, 156, 1)
green_color=rgba(16, 120, 15, 1)
```

I hope this gets implemented as it would make my screenshots look a lot nicer with a custom palette.

**Comment by voronind-com** [comment]
> Hey, I wanted to add to this as well, the merge in [#135](https://github.com/jtheoof/swappy/pull/135) appears to allow us to set the _custom_ colour but this issue in particular is to request changing the default RGB values or perhaps being able to define a custom palette.
> 
> From a configuration perspective, I'd imagine we could set the following values:
> 
> ```
> [Default]
> red_color=rgba(238, 22, 31, 1)
> blue_color=rgba(0, 66, 156, 1)
> green_color=rgba(16, 120, 15, 1)
> ```
> 
> I hope this gets implemented as it would make my screenshots look a lot nicer with a custom palette.

I use this part of my patch https://git.voronind.com/voronind/nix/src/commit/b469de2c761a9c792d4728d0033d59a41f0cae8e/patch/swappy/default-color.patch#L38-L47

Replace @accent@ with desired values.

It applies the color selected on launch only. So if you press the R key, you'll get old fashioned red etc.

Other parts of that patch change default RGB values. I use red, black and white.
